mongo/client.go

```diff
@@ -307,6 +311,11 @@ func (c *Client) Disconnect(ctx context.Context) error {
 	if disconnector, ok := c.deployment.(driver.Disconnector); ok {
 		return replaceErrors(disconnector.Disconnect(ctx))
 	}
+
+	if c.httpClient == internal.DefaultHTTPClient {
+		c.httpClient.CloseIdleConnections()
```

Comment:
I believe our Go 1.10 build is failing for this PR because `CloseIdleConnections()` is [not available](https://pkg.go.dev/net/http#Client.CloseIdleConnections) on the `http.Client` until Go 1.12. Our current minimum Go driver version is 1.10 (we're investigating bumping to 1.13 with GODRIVER-2174). Is it possible to call `CloseIdleConnections` on the `http.Transport` instead?

Reply:
okay that's fixed